src/scribe_data/check/check_language_metadata.py

```diff
@@ -0,0 +1,231 @@
+import difflib
+import json
+from pathlib import Path
+import sys
+
+LANGUAGE_DATA_EXTRACTION_DIR = Path(__file__).parent.parent / "language_data_extraction"
+
+LANGUAGE_METADATA_FILE = (
+    Path(__file__).parent.parent / "resources" / "language_metadata.json"
+)
+
+DATA_TYPE_METADATA_FILE = (
+    Path(__file__).parent.parent / "resources" / "data_type_metadata.json"
+)
```

Comment:
@catreedle you can get these from `src/scribe_data/cli/cli_utils.py` - they already exist there